internal/compiler-bridge/src/main/scala/xsbt/CompilerInterface2.scala

```diff
@@ -22,7 +22,7 @@ import java.io.File
 /**
  * This is the entry point for the compiler bridge (implementation of CompilerInterface)
  */
-final class CompilerInterface extends CompilerInterface2 {
+final class CompilerInterface2 extends xsbti.compile.CompilerInterface2 {
```

Comment:
If we're going to rename this, let's rename it to CompilerBridge, then.

Reply:
Good call. It's better than `xsbt` vs `xsbti` difference.

Reply:
Addressed in ce80b80